services/api-gateway/src/simcore_service_api_gateway/endpoints_user.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends, Security
+
+from .auth import get_current_active_user, get_current_user
+from .schemas import User
+
+router = APIRouter()
+
+
+@router.get("/users/me/", response_model=User)
+async def read_users_me(current_user: User = Depends(get_current_active_user)):
+    return current_user
+
+
+@router.get("/users/me/projects/")
+async def list_own_projects(
+    current_user: User = Security(get_current_active_user, scopes=["projects"])
+):
+    return [{"project_id": "Foo", "owner": current_user.username}]
+
+
+@router.get("/status/")
+async def read_system_status(current_user: User = Depends(get_current_user)):
+    print(current_user)
+    return {"status": "ok"}
```

Comment:
I wish there was a

@sio.event("MouseInput")
async def mouse_input(mouse_coords: MouseCoords, modifiers: MouseModifiers):

and automatic conversion into asynapi specs...

Reply:
I can imaging the way of doing that... :-) We can check some other time online